content-services/latest/support/index.md

```diff
@@ -72,7 +72,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Alfresco Search and Insight Engine 2.0.8 and later | Search and Insight Engine is compatible with Java 11 as long as you run Zeppelin in a Java 8 runtime. You can do this either in a VM or separate Java 8 based server. |
 | Alfresco Search Services 2.0.8 and later | |
 | Identity Service 2.0 | |
-| Alfresco Intelligence Services 1.5 | |
+| Alfresco Intelligence Services 3.0 | |

```

Comment:
@gionn Thanks for the suggestion. I'm guessing that the [alfresco-ai-docker-engine](https://quay.io/repository/alfresco/alfresco-ai-docker-engine) Docker image version number differs from the released Alfresco Intelligence Services (AIS) 🤯 . The latest release version of AIS is still **1.5** in [Hyland Community](https://community.hyland.com/Products/alfresco/release-notes/release-notes/Alfresco-Intelligence-Services-Releases).

@mstrankowski Do you know in which version of AIS the `3.0.0` image should be added?